langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -18,6 +18,19 @@ import ballerina/jballerina.java;
 
 # Returns the sum of zero or more decimal values.
 #
+# ```ballerina
+# // Take the sum of multiple decimal numbers.
+# decimal total = decimal:sum(1.2, 2.3, 3.4);
+#
+# // Take the sum of existing decimal number list.
+# decimal[] scores = [11.1, 22.2, 33.3];
+# decimal totalScore = decimal:sum(...scores);
+#
+# // Take the sum of multiple decimal numbers with an existing decimal number.
```

Comment:
Some comments for the float langlib are applicable here also. E.g., https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1037781532